Create ArtnetEtherENC.h (#79)
hideakitai/ArtNet: README.md

- ESP32 (Ethernet and ETH)
- ESP8266
- Almost all platforms without WiFi

> - Anyplatform supported by ENC28J60 (please read following section)

Reviewer: typo
```suggestion
- Any platform supported by ENC28J60 (please read following section)
```

- ESP32 (Ethernet and ETH)
- ESP8266
- Almost all platforms without WiFi

> - Raspberry Pi Pico with ENC28J60 (please read following section)

Reviewer: All platforms that can use EtherENC can be supported, I think.

Author: You're right,

I'll check the library documentation

- You can set Net (0-127) and Sub-Net (0-15) like `artnet.begin(net, subnet)`
- Universe (0-15) can be set in `artnet.subscribe(universe, callback)`,
- These universes (targets of the callbacks) are reflected to `net_sw` `sub_sw` `sw_in` in `ArtPollreply` automatically

> PortTypes, GoodInput/Output, SwIn, etc., are limited to 4 ports. If more than 4 callbacks are needed, it is better to subscribe to more than four and make public 4 of them

Reviewer: ```suggestion
PortTypes, GoodInput/Output, SwIn, etc., are limited to 4 ports. Only the first four ports are reflected if you subscribe to more than four callbacks.
```
